src/components/organisms/TimingEvent/TimingEvent.tsx

```diff
@@ -16,7 +17,7 @@ import { TimingEventModal } from "../TimingEventModal";
 
 export type TimingEventProps = {
   event: WorldEvent;
-  space?: SpaceWithId;
+  space?: SpaceWithId | WithId<AnyVenue>;
```

Comment:
The type of `SpaceWithId` is widely used, how many places will have to be changed if `WithId<AnyVenue>` isn't added?
Alternatively, we can define `SpaceWithStringId = WithId<AnyVenue>` .

The above can be declared OOS, no problem

Reply:
`WithId<AnyVenue>` is widely used in many of our hooks and that's the problem right here. I will try to cast it `as SpaceWithId` but if it doesn't work, I'll just leave it as it is.